config/locales/en/course/assessment/submission/submissions.yml

```diff
@@ -61,9 +61,7 @@ en:
             question: 'Question'
             question_grader: 'Grader'
             grade_summary: 'Grade Summary'
-            publish_grade_hint: >
-              The grades are currently not viewable to the student. You can publish the grades at
-              the submissions page of current assessment.
+            publish_grade_hint: 'The grades are currently not viewable to the student.'
```

Comment:
Yup, the first quoted string in the previous comment. The 2nd line was an explanation.